src/components/dialog/DialogBase.js

```diff
@@ -24,6 +24,7 @@ export const DialogBase = ({
   size,
   initialFocusRef,
   dragHandleRef,
+  ...rest
```

Comment:
I'm generally not a fan of just spreading all properties down, especially if there's only one property needed, can we just spread the element property for now?

Reply:
Let's do that, because I don't see the value of ever using a different element for a `Dialog`

Reply:
I had that at first but I also needed to pass the `onSubmit` function to make it work.
Since that's kind of a random property to only applies to the `form` element I preferred just spreading all the props